src/main/java/com/treasuredata/android/TreasureData.java

```diff
@@ -846,6 +863,34 @@ public void fetchUserSegments(List<String> profilesTokens, Map<String, String> k
         }
     }
 
+    private String defaultValueTableKey(String database, String table) {
+        String _database = database == null ? "" : database;
+        String _table = table == null ? "" : table;
+        return String.format("%s.%s", _database, _table);
+    }
+
+    public void setDefaultValue(Object value, String key, String database, String table) {
+        if (defaultValues == null) defaultValues = new HashMap();
+        String tableKey = this.defaultValueTableKey(database, table);
+        if (!defaultValues.containsKey(tableKey)) defaultValues.put(tableKey, new HashMap());
+        Map tableMap = defaultValues.get(tableKey);
+        tableMap.put(key, value);
+    }
+
+    public Object defaultValue(String key, String database, String table) {
```

Comment:
If there is no particular reason (consistent naming with iOS perhaps?), may I suggest rename this to a Java canonical style: `getDefaultValue`?

Reply:
Let's go with `getDefaultValue`. td-android-sdk and td-ios-sdk does not have 100% API naming consistency anyway :)